docs/2_development.md

```diff
@@ -183,7 +184,8 @@ below, with the relevant credentials and key values replaced.
   "twitterApiCredentials": {
     "appKey": "{TWITTER_APP_KEY}",
     "appToken": "{TWITTER_APP_TOKEN}",
-    "bearerToken": "{TWITTER_APP_BEARER_TOKEN}"
+    "bearerToken": "{TWITTER_APP_BEARER_TOKEN}",
+    "useEssentialOrElevatedV2": true
```

Comment:
Do you need `useEssentialOrElevatedV2: true` if you have access to **Full-Archive Search**? I think this should go into it's own section.

Reply:
@kilemensi Not sure what you mean by its own section

Reply:
In the doc @kelvinkipruto, there is already:

1. **If using the Enteprise Full-Archive Search API:**
2. **If using the v2 Full-Archive Search endpoint:**

You are trying to squeeze `useEssentialOrElevatedV2` into `2` above. I'm saying, shouldn't that go into its own section: **If using the v2 Essential or Elevated API:**?